tsconfig.test.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "@octokit/tsconfig",
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "declaration": true,
+    "outDir": "pkg/dist-types",
+    "emitDeclarationOnly": true,
+    "sourceMap": true,
+    "verbatimModuleSyntax": false
+  },
+  "include": ["src/**/*"]
```

Comment:
```suggestion
  "include": ["test/**/*"]
```

Reply:
This one I'm less sure about.

Reply:
I don't think we should take this: then the tsconfig would only apply to the single test file, not the source code. Perhaps `include` should be something like `**/*` if we want to include both the test file(s) and the source code in the test tsconfig.